CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Improvements:
 
 ## Bug fixes:
+Update Microsoft.Extensions.DependencyModel to fix security voulnerable System.Text.Json 8.0.4 as dependency
```

Comment:
Could you please format the entry according to the style you see below. For fixes this is:

`* Fix: <problem description> (#<issue or pr number>)`

Could you please also add yourself to the contributors list below as well?